frontend/src/pages/DegreeWizard/steps/PreviousCoursesStep.jsx

```diff
@@ -4,7 +4,7 @@ import { useSelector, useDispatch } from "react-redux";
 import { plannerActions } from "../../../actions/plannerActions";
 import { useNavigate } from "react-router-dom";
 import { EditOutlined } from "@ant-design/icons";
-import DebouncingSelect from "./DebouncingSelect";
+import DebouncingSelect from "./MultiSelectCourse";
```

Comment:
im too small-brain to figure out how this import actually worked, but since it works i wont question it lol

Reply:
oh yea, that's so weird. I have no idea why that works 🤨. But anyway, I've updated it now. Good catch!